database.aml

```diff
@@ -1,224 +1,327 @@
+  organization_id int | Id of the organization
+  name varchar(255) | Name of the category
+fk purchase_categories.organization_id -> organizations.id
+
+
+#=========# INCOMES #=========#
+
+incomes | Represents an income
+  code varchar(255) pk | Code of the income
+  organization_id int | Id of the organization
+  transaction_code varchar(255) nullable | Id of the transaction, null if the income is not processed
+  tax_transaction_code varchar(255) nullable | Id of the transaction, null if the income is not processed
+  created_at date | Date of creation
+fk incomes.organization_id -> organizations.id
+fk incomes.transaction_code -> internal_transactions.code
+fk incomes.tax_transaction_code -> internal_transactions.code
+
+
+#=========# PERSONAL #=========#
+
+persons | Any person
+  id int pk | Id of the person
+  organization_id int | Id of the organization
+  first_name varchar(255) | First name of the person
+  last_name varchar(255) | Last name of the person
+  email varchar(255) nullable | Email of the person
+  created_at date | Date of creation
+  updated_at date | Date of last update
+fk persons.organization_id -> organizations.id
+
+users | Any user
+  id int pk | Id of the user
+  organization_id int | Id of the organization
+  email varchar(255) | Email of the user
+  username varchar(255) | Username of the user
+  password varchar(255) | Password of the user
+  validation_code varchar(255) nullable | Validation code of the user, null if the user is validated
+  is_organization_admin boolean = false | Is the user is the admin account of an organization
+  group_id int = 1 | Id of the group, 1 for default group
+  specific_permissions varchar(255) nullable | Specific permissions of the user
+  avatar_id int = 1 | Id of the picture of the user
+  created_at date | Date of creation
+  updated_at date | Date of last update
+fk users.organization_id -> organizations.id
 fk users.group_id -> groups.id
-fk users.organization_id -> organization.id
+fk users.avatar_id -> pictures.id
 
+groups | Represents a group
+  id int pk | ID of the group
+  name varchar(255) | Name of the group
+  permissions text | Permissions of the group (serialized array)
+  organization_id int nullable | ID of the organization of the group, null if the group is global
+  created_at date | Indicate when the group has been created
+  updated_at date | Indicate when the group has been updated
+fk groups.organization_id -> organizations.id
+
+personal_accounts | Accounts for a person
+  id int pk | Id of the account
+  amount number = 0 | Amount of money in the account
+  person_id int | Id of the person
+  card_id varchar(255) | Id of the student card QR code
+  created_at date | Date of creation
+  updated_at date | Date of last update
+fk personal_accounts.person_id -> persons.id
+
+personal_transactions | Transaction on a personal account
+  id int pk | Id of the transaction
+  debit int = 0 | Debit amount
+  credit int = 0 | Credit amount
```

Comment:
This field can be turned to `credit_amount`, it would be more undersable
/!\ It's not an integer because of a money amount. So turn it to Float instead